schemas/manifest-localization/schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "PiWebImportPluginManifestLocalizationSchema",
+  "title": "PiWeb import plugin manifest localization schema",
+  "description": "Describes the localization file for a PiWeb import plugin manifest.",
+  "type": "object",
+  "properties": {
+    "$schema": {
+      "type": "string"
+    },
+    "translation": {
+      "type": "object",
+      "description": "The list of localizations for the plugin",
+      "properties": {
+        "title": {
+          "description": "The localization for the plugin title",
+          "type": "string"
+        },
+        "description": {
+          "description": "The localization for the plugin description",
+          "type": "string"
+        },
+        "homepage": {
+          "description": "The localization for the plugin homepage",
+          "type": "string"
+        },
+        "contact": {
+          "description": "The localization for the plugin contact",
+          "type": "string"
+        },
+        "documentation": {
+          "description": "The localization for the plugin documentation",
+          "type": "string"
+        },
+        "<importAutomationId>.title": {
+          "description": "The localization for the plugin import automation title",
+          "type": "string"
+        },
+        "<importAutomationId>.description": {
+          "description": "The localization for the plugin import automation description",
+          "type": "string"
+        }
```

Comment:
Are we ok with that syntax?
E.g. `<importAutomationId>.title`?

There seems to be something like "pattern properties", maybe we should use this instread?
https://json-schema.org/understanding-json-schema/reference/object#patternProperties

Reply:
Since we changed the provides object to directly present the provided module, we probably should just attribute the title and the summery to this provides object. This probably needs a code change in the AutoImporter though.

Reply:
Oh, here is something mixed up. I guess the latest implementation is to use:
`importAutomation.displayName`
`importAutomation.summary`
`importFormat.display`
So there is already a fixed ID for these keys.

Reply:
Yeah but I like the provides.summary and provides.displayName variant better.